kafka/sync_producer.go

```diff
@@ -0,0 +1,43 @@
+package kafka
+
+import (
+	"fmt"
+
+	"github.com/Shopify/sarama"
+	"github.com/sirupsen/logrus"
+)
+
+type SyncProducer struct {
+	conn sarama.SyncProducer
+}
+
+func InitSyncProducer(config *sarama.Config, brokers []string) (*SyncProducer, error) {
```

Comment:
use custom config here so it's not forcing client code to import `sarama` package

Reply:
I agree with your point. However, if you look at the sarama.Config struct, you will see how many fields and structured fields there. Do we really need to copy it? Maybe let's keep sarama dependcy?

Reply:
I made an init func that returns a default sarama config.